src-docs/src/views/flex/flex_item.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { EuiButton } from '../../../../src/components/button';
+import { EuiFlexGroup, EuiFlexItem } from '../../../../src/components/flex';
+
+export default () => (
+  <EuiFlexGroup>
+    <EuiFlexItem>
+      <EuiButton>Buttons will widen</EuiButton>
```

Comment:
```suggestion
      <EuiButton color="primary">Buttons will widen</EuiButton>
```

In amsterdam, these blend into the background helper. Making them primary will help them read better.

Reply:
Did you mean add `fill` instead? They already default to the primary color :)

Reply:
Sheesh. Yes. Updated both.